native/cocos/renderer/pipeline/Define.h

```diff
@@ -48,7 +48,10 @@ class RenderFlow;
 // Skinning models with number of bones more than this capacity will be automatically switched to texture skinning.
 // But still, you can tweak this for your own need by changing the number below
 // and the JOINT_UNIFORM_CAPACITY macro in cc-skinning shader header.
-constexpr int JOINT_UNIFORM_CAPACITY = 30;
+class CC_DLL SkinningJointCapacity {
+public:
+    static uint jointUniformCapacity;
+};
```

Comment:
Why change to use class?

Reply:
Got it, it is not a const value. I think can just use extern as `extern CC_DLL uint32_t globalSet;`, don't need a class.